package.json

```diff
@@ -24,7 +24,10 @@
     "build": "webpack --config webpack.config.js --mode=development",
     "bundle": "webpack --config webpack.config.js --mode=production",
     "copy": "cp package.json dist/ && cp README.md dist/",
-    "production": "npm run bundle && npm run copy"
+    "production": "npm run bundle && npm run copy",
+    "lic_direct": "npx adsk-npm-license-puller --path . --app-name 'NotificationCenter' --verbose --about-box ./license_output/about-box_direct.html --about-box-type desktop --year 2022 --paos ./license_output/paos_direct.csv",
```

Comment:
Can we can provide `--yes` option to `npx`?
`"lic_direct": "npx --yes @adsk\adsk-npm-license-puller ...",`
`"lic_transitive": "npx --yes @adsk\adsk-npm-license-puller ...",`
`"generate_license": "npm run lic_direct && npm run lic_transitive"`

Reply:
hmm, it's a good point @avidit - I forgot with NPX we probably don't need the manual install step at all, for now I think leaving `--yes` off is ok - theres arguments for and against it (security/package squatting) - but I am fighting with Dynamo's UI at the moment, will get back to this after since these prs can really wait until after 2.16 I believe since I have the html content.